### PR TITLE
More mobile site

### PR DESCRIPTION
**Pull request by klahnakoski**
add more mobile sites

### CODE REVIEW

File: src/quantum/config.js [hunk 1]
```diff
@@ -1,66 +1,76 @@
...
+    //   },
+    // },
+    // {
+    //   title: 'JetStream',  //RUN ON JS BENCH
+    //   secondLink: '/quantum/windows10-64/JetStream',
+    //   secondTitle: 'Breakdown',
+    //   signatures: {
+    //     'windows10-64': '799db0d801c7248924e93ca5a0b770020498da69',
+    //   },
+    // },
```
Comment (armenzg) [inline src/quantum/config.js]: Please remove commented out code.

File: src/quantum/config.js [hunk 2]
```diff
@@ -1,66 +1,76 @@
...
+    //   secondLink: '/quantum/windows7-32/ARES6',
+    //   secondTitle: 'Breakdown',
+    //   signatures: { 'windows7-32': '9585321f110f16414e272956c8892a38d03e7797' },
+    // },
+    // {
+    //   title: 'JetStream',  //RUN ON JS BENCH
+    //   secondLink: '/quantum/windows7-32/JetStream',
+    //   secondTitle: 'Breakdown',
+    //   signatures: { 'windows7-32': '95535e0278558bd59ae324920ca0469278faadb4' },
+    // },
```
Comment (armenzg) [inline src/quantum/config.js]: Same here.

**Comment by armenzg** [comment]
There's also some issues with some of the benchmarks (see console):
https://deploy-preview-337--firefox-health-dashboard.netlify.com/android/tp6m?test=loadtime&platform=android-hw-g5-7-0-arm7-api-16

File: src/quantum/config.js [hunk 3]
```diff
@@ -179,10 +189,31 @@ const SUITES = {
...
+    ['geckoview', 'Tp6 mobile: Bing Restaurants', "raptor-tp6m-bing-restaurants-geckoview"],
+    ['geckoview', 'Tp6 mobile: Booking', "raptor-tp6m-booking-geckoview"],
+    ['geckoview', 'Tp6 mobile: CNN', "raptor-tp6m-cnn-geckoview"],
+    ['geckoview', 'Tp6 mobile: CNN AmpStories', "raptor-tp6m-cnn-ampstories-geckoview"],
+    ['geckoview', 'Tp6 mobile: Kleinanzeigen', "raptor-tp6m-ebay-kleinanzeigen-geckoview"],
+    ['geckoview', 'Tp6 mobile: Kleinanzeigen Search', "raptor-tp6m-ebay-kleinanzeigen-search-geckoview"],
+    ['geckoview', 'Tp6 mobile: ESPN', "raptor-tp6m-espn-geckoview"],
+    ['geckoview', 'Tp6 mobile: Facebook', 'raptor-tp6m-facebook-geckoview'],
     ['geckoview', 'Tp6 mobile: Google', 'raptor-tp6m-google-geckoview'],
+    ['geckoview', 'Tp6 mobile: Google Mpas', "raptor-tp6m-google-maps-geckoview"],
```
Comment (davehunt) [inline src/quantum/config.js]: typo Maps

**Comment by klahnakoski** [comment]
> 
> 
> There's also some issues with some of the benchmarks (see console):
> https://deploy-preview-337--firefox-health-dashboard.netlify.com/android/tp6m?test=loadtime&platform=android-hw-g5-7-0-arm7-api-16

Yes, the benchmark PR is fixing that.  I merged that PR into this one:  Review this one after that lands to see the true diff

**Comment by klahnakoski** [comment]
> 
> 
> It seems you have two different pieces of work in the same PR.
> Could you please separate them? It makes it hard to review.

This is only one piece of work.  The `framework` code depends on configuration that guides it.  The configuration depends on the code to show the right thing.

**Comment by klahnakoski** [comment]
closing this, to open one branched off of `bench`

**Pull request closed without merging**